docs/requirements.txt

```diff
@@ -0,0 +1,7 @@
+mkdocs==1.5.0
+mkdocs-material==8.2.6
+mkdocs-spellcheck==0.2.1
```

Comment:
It doesn't seem like we actually use `mkdocs-spellcheck` as it's not listed in our `mkdocs.yml` [as a plugin](https://github.com/pawamoy/mkdocs-spellcheck/tree/1.0.2?tab=readme-ov-file#usage)
```suggestion
```